### ISSUE TITLE
Configuration fails if "24h" zone 

### ISSUE THREAD

**Issue by mced76** [petrleocompel/hikaxpro_hacs#43]
Hi!
Thanks for this integration and the fork it's very good.
I have a trouble if my AX-Pro sensor are in "24h" zone. Configuration fails
In the log I have
```
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 387, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/config/custom_components/hikvision_axpro/__init__.py", line 105, in async_setup_entry
    await hass.async_add_executor_job(coordinator.init_device)
  File "/usr/local/lib/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/hikvision_axpro/__init__.py", line 177, in init_device
    self.load_devices()
  File "/config/custom_components/hikvision_axpro/__init__.py", line 181, in load_devices
    devices = self._load_devices()
  File "/config/custom_components/hikvision_axpro/__init__.py", line 194, in _load_devices
    return ZonesConf.from_dict(response.json())
  File "/config/custom_components/hikvision_axpro/model.py", line 823, in from_dict
    list = from_list(ListElement.from_dict, obj.get("List"))
  File "/config/custom_components/hikvision_axpro/model.py", line 29, in from_list
    return [f(y) for y in x]
  File "/config/custom_components/hikvision_axpro/model.py", line 29, in <listcomp>
    return [f(y) for y in x]
  File "/config/custom_components/hikvision_axpro/model.py", line 807, in from_dict
    zone = ZoneConfig.from_dict(obj.get("Zone"))
  File "/config/custom_components/hikvision_axpro/model.py", line 659, in from_dict
    zone_type = ZoneType(obj.get("zoneType"))
  File "/usr/local/lib/python3.10/enum.py", line 385, in __call__
    return cls.__new__(cls, value)
  File "/usr/local/lib/python3.10/enum.py", line 710, in __new__
    raise ve_exc
ValueError: '24h' is not a valid ZoneType
```

If I change my sensor in another zone, AX integration works and it is awesome !

Thanks a lot for your job

Regards

**Comment by petrleocompel** [comment]
Please what version are you using. It is very importnant. Are you using latest beta of version 2 ? or stable version 1 ?

**Comment by mced76** [comment]
Yes, I Use the latest beta 2.x

**Comment by petrleocompel** [comment]
Okay.. Hate when this happens.
It is undocumented type...
In docs
```json5
{
/*optional, string,
zone type:
 - "Instant"-instant zone,
 - "Delay"-delay zone, "Follow"-follow zone, 
 - "Perimeter"-perimeter zone, 
 - "24hNoSound"-24-hour silent zone, 
 - "Emergency"-panic zone, 
 - "Fire"-fire zone, 
 - "Gas"-gas zone, 
 - "Medical"-medical zone, 
 - "Timeout"-timeout zone, 
 - "Non-Alarm"-disabled zone, 
 - "Key"-key zone
*/
zoneType: "",
}
```

**Comment by petrleocompel** [comment]
@mced76 Fixed, beta10 released + beta thread -> #41

**Comment by mced76** [comment]
Yes, it works!
Thanks
